x-pack/plugins/security_solution/public/common/components/text_field_value/index.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiToolTip } from '@elastic/eui';
+import React from 'react';
+
+const trimTextOverflow = (text: string, maxLength?: number) => {
+  if (maxLength !== undefined && text.length > maxLength) {
+    return `${text.substr(0, maxLength)}...`;
+  } else {
+    return text;
+  }
+};
+
+interface Props {
+  fieldName: string;
+  value: string;
+  maxLength?: number;
+  className?: string;
+}
+
+export const TextFieldValue = ({ fieldName, value, maxLength, className }: Props) => {
```

Comment:
can you maybe add a comment describing when someone would want to use this component?

Reply:
Added.